crowdin.yml

```diff
@@ -0,0 +1,8 @@
+project_id: "620834" # Your Crowdin project ID
+api_token: "57722329698b7884c01038b1a041700c43bd70527d4859677b104ac6aeedbf21eb83cb94a3d26672"
```

Comment:
These should be read from `env`. We should do it before merge.

Reply:
Done